src/ssl.c

```diff
@@ -7027,12 +7027,31 @@ int wolfSSL_CTX_load_verify_locations_ex(WOLFSSL_CTX* ctx, const char* file,
         /* pass directory read failure to response code */
         if (fileRet != WC_READDIR_NOFILE) {
             ret = fileRet;
+    #if defined(WOLFSSL_QT)
+            if (ret == BAD_PATH_ERROR && 
+                flags & WOLFSSL_LOAD_FLAG_IGNORE_BAD_PATH_ERR) {
+           /* QSslSocket always loads certs in system folder 
+            * when it is initialized.
+            * Compliant with OpenSSL when flag sets.*/
+                ret = WOLFSSL_SUCCESS;
+            }
+            else
```

Comment:
Please add brackets around else body here.

Reply:
Add brackets and moved comments above in 4 spaces